src/App.tsx

```diff
@@ -42,19 +42,19 @@ const App: React.FC = () => (
                 </IonRouterOutlet>
                 <IonTabBar slot="bottom">
                     <IonTabButton tab="eventsist" href="/eventsList">
-                        {<IonIcon icon={list} />}
-                        <IonLabel>Event list</IonLabel>
+                        <IonIcon src="assets/icon/tabs/icon-calendar-full.svg" />
```

Comment:
Don't hardcode paths here. You can import images - https://create-react-app.dev/docs/adding-images-fonts-and-files/ :)